app/src/main/java/org/mozilla/focus/utils/Settings.java

```diff
@@ -115,4 +116,17 @@ public void setDefaultSearchEngine(SearchEngine searchEngine) {
     /* package */ String getPreferenceKey(int resourceId) {
         return resources.getString(resourceId);
     }
+
+    /**
+     * clear cache and return clear msg
+     * @param context
+     * @return clearCache Msg
+     */
+    public String clearCache(Context context) {
```

Comment:
I'm also thinking if we should change to static method if we don't need the class instance.